SPECS/kata-containers/kata-containers.spec

```diff
@@ -21,21 +21,16 @@
 %global kataclhdir              /usr/share/cloud-hypervisor
 %global katainitrddir           /var/cache/kata-containers/osbuilder-images/kernel-uvm
 
-%global runtime_make_vars       QEMUPATH=%{qemupath} \\\
-                                KERNELTYPE="compressed" \\\
+%global runtime_make_vars       KERNELTYPE="compressed" \\\
                                 KERNELPARAMS="systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0" \\\
-                                DEFSHAREDFS="virtio-fs" \\\
-                                DEFVIRTIOFSDAEMON=%{_libexecdir}/"virtiofsd" \\\
-                                DEFVIRTIOFSCACHESIZE=0 \\\
-                                DEFSANDBOXCGROUPONLY=false \\\
+                                DEFVIRTIOFSDAEMON=%{_libexecdir}/"virtiofsd-rs" \\\
+                                DEFSTATICRESOURCEMGMT_CLH=true \\\
                                 DEFSTATICSANDBOXWORKLOADMEM=1792 \\\
                                 DEFMEMSZ=256 \\\
                                 SKIP_GO_VERSION_CHECK=y \\\
-                                MACHINETYPE=%{machinetype} \\\
                                 DESTDIR=%{buildroot} \\\
                                 PREFIX=/usr \\\
-                                FEATURE_SELINUX="yes" \\\
-                                DEFENABLEANNOTATIONS=['\\\".*\\\"'] \\\
+                                # Make configuration.toml point to the CH config
```

Comment:
I like this comment!

Reply:
I just removed it because I think that broke the build :( I can readd it before the statement

Reply:
Sounds good. I was always wondering how this piece about the config was stitched together, so the comment was very helpful

Reply:
Added back in before the global var declaration.